DadMakeAFunny/app/src/main/java/com/example/dadmakeafunny/Joke.java

```diff
@@ -0,0 +1,46 @@
+package com.example.dadmakeafunny;
+
+/*
+Joke class containing all the jokes as strings
+ */
+
+public class Joke {
+
+    private String title;
+    private String text;
+    private String link;
+
+    public Joke(){
```

Comment:
We need another constructor that takes in the title, text and link as parameters. 

Something like `Joke(String title, String text, String link)`